pkgs/development/python-modules/sismic/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  behave,
+  ruamel-yaml,
+  schema,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "sismic";
+  version = "1.6.6";
+  build-system = [ setuptools ];
+
+  src = fetchFromGitHub {
+    owner = "AlexandreDecan";
+    repo = "sismic";
+    rev = "refs/tags/${version}";
+    hash = "sha256-MvJyyERH0l5547cVmpxnHXRf9q1ylK9/ZfyLYBQfsbY=";
+  };
+
+  # Remove broken tests "fixture 'mocker' not found"
+  postPatch = ''
+    rm tests/test_bdd.py
+    rm tests/test_code.py
+    rm tests/test_clock.py
+    rm tests/test_runner.py
+    rm tests/test_property.py
+  '';
```

Comment:
You need to add `pytest-mock` to `nativeCheckInputs`.
Otherwise, use `disabledTestPaths` instead.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#common-issues-common-issues

Reply:
Fixed.